docs/user-guide/garbage-collector.md

```diff
@@ -0,0 +1,21 @@
+---
+---
+
+* TOC
+{:toc}
+
+## WARNING: Garbage Collector is an alpha feature, use it at your own risk!
+
+### What does the garbage collector do
+
+The garbage collector (GC) monitors the cluster and checks the metadata.ownerReferences field of each object. If all the owners present in metadata.ownerReferences do not exist in the cluster, the GC will request the API server to delete the object.
```

Comment:
Thanks. I'll reword it as "If none of the owners present in metadata.ownerReferences exists in the cluster, the GC will delete the object."

"If any of the owners..." is not accurate.